crypto/src/key/hdkd/child_number.rs

```diff
@@ -61,6 +61,10 @@ impl ChildNumber {
         }
     }
 
+    pub fn to_encoded_be_bytes(self) -> [u8; 4] {
```

Comment:
into, instead of to.

You can also implement the From trait as an alternative solution, which will implement Into for you.

Reply:
Btw, same applies to the function above